crates/marketplace/src/service.rs

```diff
@@ -41,6 +42,28 @@ pub use marketplace_builder_shared::utils::EventServiceStream;
 
 use crate::hooks::BuilderHooks;
 
+/// Configuration to initialize the builder
+#[derive(Debug, Clone)]
+pub struct BuilderConfig<Types: NodeType> {
```

Comment:
Sneaked in a config for the service instead of passing a bajilion parameters to the constructor